src/sql/base/browser/ui/chart/chart.component.html

```diff
@@ -1,5 +1,5 @@
 <div style="display: block; width: 100%; height: 100%; position: relative;">
 	<div class="chart-container">
-		<canvas id="MyChart" tabindex="0">{{ _chart }}</canvas>
+		<canvas id="MyChart" tabindex="0"></canvas>
```

Comment:
This ID should be something more... intentional.  Can it be a configurable property, or automatically pull from some known ID?